dll/shellext/shellbtrfs/CMakeLists.txt

```diff
@@ -18,16 +20,19 @@ list(APPEND SOURCE
     scrub.cpp
     send.cpp
     volpropsheet.cpp
-    ${CMAKE_CURRENT_BINARY_DIR}/shellbtrfs.def)
+    precomp.h)
 
-include_directories(
-    ${REACTOS_SOURCE_DIR}/drivers/filesystems/btrfs)
+add_library(shellbtrfs SHARED
+    ${SOURCE}
+    guid.c
```

Comment:
Why not in the source list?
Was that changed mandatory btw? (Getting guid.c)

Reply:
Yeap, it's mandatory because this is how we handle GUIDs with modules that have PCH. Furthermore, the guid source file should not be among the list that uses the PCH, as per the uppercase warning we include in all those guid source files in our codebase.

Reply:
OK, thanks.